<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro</class>
 <widget class="QMainWindow" name="Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>831</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
     <item>
      <widget class="QFrame" name="frame_superior">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>683</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bt_minimizar">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>C:/Users/frangarayco/Pictures/guion.jpg</normaloff>C:/Users/frangarayco/Pictures/guion.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_cerrar">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>C:/Users/frangarayco/Pictures/cerrarPiola.jpg</normaloff>C:/Users/frangarayco/Pictures/cerrarPiola.jpg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_inferior">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(61,61,61);
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
color: rgb(255,255,255);
font: 77 10pt &quot;Arial Black&quot;;
}
color: rgb(0,206,151);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-10</y>
          <width>301</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
font: 12pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>INGRESAR DATOS PARA CREAR CUENTA: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>Apellido:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>Correo electronico:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>Fecha de nacimiento: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>Nombre de usuario: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>Contraseña:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>290</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>Repetir contraseña: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_nombre">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_apellido">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_user">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_mail">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>170</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_nacimiento">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>210</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_contra">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>250</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_repetir">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>290</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_repetir">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>290</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string>Mostrar contraseña</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_mostrar">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>250</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string>Mostrar contraseña</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_registrar">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>320</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string>Registrarme</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>370</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
        <property name="text">
         <string>¿Desea crear una contraseña aleatoria? Esta es mas segura</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_genera">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>380</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string>GENERAR</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_copiar">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>420</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string>COPIAR</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="text_aleatorio">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>420</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);
</string>
        </property>
       </widget>
       <widget class="QLabel" name="signal_registro">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>460</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="signal_nacimiento">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>210</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="signal_user">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>130</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="signal_nacimiento_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>210</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,206,151);</string>
        </property>
        <property name="text">
         <string>dd-mm-aaaa</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
